docs/babel.config.js

```diff
@@ -1,8 +1,8 @@
 const bpmr = require('babel-plugin-module-resolver');
 const fse = require('fs-extra');
-const path = require('path');
+// const path = require('path');
 
-const errorCodesPath = path.resolve(__dirname, './public/static/error-codes.json');
+// const errorCodesPath = path.resolve(__dirname, './public/static/error-codes.json');
```

Comment:
why do we need the commented code? Should we remove it?

Reply:
removed